roles/kubernetes/master/tasks/kubeadm-setup.yml

```diff
@@ -111,75 +105,76 @@
     PATH: "{{ bin_dir }}:{{ ansible_env.PATH }}"
   notify: Master | restart kubelet
 
-- name: set kubeadm certificate key
-  set_fact:
-    kubeadm_certificate_key: "{{ item | regex_search('--certificate-key ([^ ]+)','\\1') | first }}"
-  with_items: "{{ (kubeadm_init|default({'stdout_lines': []}))['stdout_lines'] }}"
-  run_once: yes
-  when:
-    - kubeadm_version is version('v1.14.0', '>=')
-    - kubeadm_certificate_key is not defined
-    - kubeadm_init is defined
```

Comment:
This would need to be `kubeadm_init.stdout_lines is defined` since kubeadm_init is defined regardless of whether the previous task runs.

Reply:
Thanks! But this is a revert. The revert is happening because the task on L136 fails

Reply:
Thanks! But this is a revert. The revert is happening because the task on L136 fails

Reply:
Sorry! github glitch